internal/cloud/realm/auth.go

```diff
@@ -96,6 +97,24 @@ func (c *client) getAuth(options api.RequestOptions) (string, error) {
 	return "", nil
 }
 
+//Refresh Auth uses the user's refresh token to attempt to create a new access token
+func (c *client) refreshAuth() (string, error) {
+	res, resErr := c.do(http.MethodPost, refreshAuthPath, api.RequestOptions{RefreshAuth: true})
```

Comment:
does this also want `PreventRefresh: true`?

Reply:
That is a great call out. I believe we don't get an InvalidSession code from this call but I am lot more comfortable adding it